spec/views/spree/admin/orders/invoice.html.haml_spec.rb

```diff
@@ -27,7 +27,10 @@
                                     display_checkout_total_less_tax: '8',
                                     outstanding_balance_label: 'Outstanding Balance'
 
-    stub_request(:get, ->(uri) { uri.to_s.include? "/css/mail" })
+    # Ignore requests for CSS pack like: 'http://test.host/packs-test/css/mail-1ab2dc7f.css'
+    VCR.configure do |config|
+      config.ignore_hosts('test.host')
+    end
```

Comment:
It looks like the rendering is trying to access CSS in test.host. Configuring VCR to ignore that request just means that it's still handled by Webmock and it doesn't allow external requests. Well, since `test.host` is not a valid domain name, it fails resolving it. The original stub solved this by intercepting the request and not returning anything. 

So when we have the stub in there, we get the frozen string error but the stack trace is limited to our code. I put a `binding.pry` in front of that line and called `wicked_pdf_stylesheet_pack_tag(source)` within the console which revealed the same error but now pointing to the source:

https://github.com/mileszs/wicked_pdf/blob/edcd80e7a96b5fe79476f7984e5ff8f3765f220d/lib/wicked_pdf/wicked_pdf_helper/assets.rb#L191-L192

Wicked PDF tries to read the URL and gets a string. But that string is provided by Webmock. It tries to force the encoding on the string which would modify it but it's not allowed to because Webmock is using a frozen string in this new version.

I tried defining the returned string but it's frozen by default because we add `frozen_string_literal: true` at the top of our files. A workaround is to duplicate it to a non-frozen string:

```rb
    stub_request(:get, ->(uri) { uri.to_s.include? "/css/mail" }).to_return(body: "".dup)
```

This works. And we don't need to ignore the host in VCR. I reckon that this should be fixed in wicked_pdf but webmock could also be patched to be compatible with this.

I think this commit introduced it:
* https://github.com/bblimke/webmock/commit/8315dc52ad352b570080b8f152d047e6dcdb9180

Maybe it's this line that could be changed:

https://github.com/bblimke/webmock/blob/63940ac95345968cf435820a62102d913441862c/lib/webmock/response.rb#L40-L42

Patching Webmock works but I'm not sure if they would accept my patch. So we should go with our workaround for now, especially because it's only needed in combination with wicked_pdf.